interface/patient_file/label.php

```diff
@@ -75,5 +75,11 @@
 	$pdf->Add_Label($text);
 }
 
-$pdf->Output();
+$output = $pdf->Output('','S');
+$output = base64_encode($output);
 ?>
+
+<title>Chart Label</title>
```

Comment:
nice catch this does the trick

Reply:
Need to check for issues with base64_encode php dependency.